### ISSUE TITLE
add_new_observations has no apparent effect on seasonal ARIMAs

### ISSUE THREAD

**Issue by animalmutch** [tgsmith61591/pmdarima#104]
Using your example code, but printing the results of predictions before and after adding new observations, I see no difference in the predictions:

```python
>>> import pmdarima as pm
>>> from pmdarima.datasets import load_wineind

>>> y = load_wineind()
>>> train, test = y[:125], y[125:]
>>> arima = pm.ARIMA(order=(1, 1, 2), seasonal_order=(0, 1, 1, 12))
>>> arima.fit(y)
>>> predictions_before_new_observations = arima.predict(n_periods=100)
>>> print(predictions_before_new_observations)
[ 21966.69719064  25983.70920537  30225.30979849  35417.12497605
  13011.36189384  19639.41029275  21506.58177734  23674.34000129
  21685.76396121  23669.77820915  26955.35258274  22755.25064259
  19808.16136752  23578.78187736  27845.86723277  32923.89430476
  10475.68782067  17024.74537797  18831.64801826  20929.54676213
  18876.02419507  20792.575179    24011.97552646  19745.03912793
  16731.45371123  20435.40477139  24635.90945687  29647.31038012
   7132.50104453  13614.94381595  15355.23778394  17386.52472372
  15266.39195688  17116.3319192   20269.12166605  15935.57445125
  12855.37832874  16492.7186265   20626.61257859  25571.40275359
   2989.98267735   9405.81470422  11079.49792966  13044.17412588
  10857.43061599  12640.759835    15726.93883867  11326.78088062
   8179.97401491  11750.70356944  15817.9867783   20696.16621008
  -1951.86460938   4397.35667427   6004.42915649   7902.49460948
   5649.14035637   7365.85883215  10385.4270926    5918.65839133
   2705.24078239   6209.3595937   10210.03205935  15021.6007479
  -7693.04081479  -1410.43027436    130.03146464   1961.4861744
   -358.47882193   1291.62891063   4244.58642786   -288.79301663
  -3568.82136879   -131.3133007    3802.74842171   8547.70636704
 -14233.54593886  -8017.54614165  -6543.69514588  -4778.85117934
  -7165.42691889  -5581.92992956  -2695.58315555  -7295.57334327
 -10642.21243865  -7271.31511379  -3403.86413459   1274.48306752
 -21573.37998161 -15423.99092763 -14016.75067508 -12318.51745176
 -14771.70393453 -13254.81768842 -10435.08165764 -15101.68258858
 -18514.93242718 -15210.64584554 -11409.80560957  -6798.06915068]

>>> arima.add_new_observations(test)  # pretend these are the new ones
>>> predictions_after_new_observations  = arima.predict(n_periods=100)
>>> print(predictions_after_new_observations)
[ 21966.69719064  25983.70920537  30225.30979849  35417.12497605
  13011.36189384  19639.41029275  21506.58177734  23674.34000129
  21685.76396121  23669.77820915  26955.35258274  22755.25064259
  19808.16136752  23578.78187736  27845.86723277  32923.89430476
  10475.68782067  17024.74537797  18831.64801826  20929.54676213
  18876.02419507  20792.575179    24011.97552646  19745.03912793
  16731.45371123  20435.40477139  24635.90945687  29647.31038012
   7132.50104453  13614.94381595  15355.23778394  17386.52472372
  15266.39195688  17116.3319192   20269.12166605  15935.57445125
  12855.37832874  16492.7186265   20626.61257859  25571.40275359
   2989.98267735   9405.81470422  11079.49792966  13044.17412588
  10857.43061599  12640.759835    15726.93883867  11326.78088062
   8179.97401491  11750.70356944  15817.9867783   20696.16621008
  -1951.86460938   4397.35667427   6004.42915649   7902.49460948
   5649.14035637   7365.85883215  10385.4270926    5918.65839133
   2705.24078239   6209.3595937   10210.03205935  15021.6007479
  -7693.04081479  -1410.43027436    130.03146464   1961.4861744
   -358.47882193   1291.62891063   4244.58642786   -288.79301663
  -3568.82136879   -131.3133007    3802.74842171   8547.70636704
 -14233.54593886  -8017.54614165  -6543.69514588  -4778.85117934
  -7165.42691889  -5581.92992956  -2695.58315555  -7295.57334327
 -10642.21243865  -7271.31511379  -3403.86413459   1274.48306752
 -21573.37998161 -15423.99092763 -14016.75067508 -12318.51745176
 -14771.70393453 -13254.81768842 -10435.08165764 -15101.68258858
 -18514.93242718 -15210.64584554 -11409.80560957  -6798.06915068]
```

Am I missing something about how add_new_observations works?

Windows-10-10.0.17763-SP0
Python 3.6.4 (v3.6.4:d48eceb, Dec 19 2017, 06:04:45) [MSC v.1900 32 bit (Intel)]
pmdarima 1.1.0
NumPy 1.15.4
SciPy 1.2.1
Scikit-Learn 0.20.3
Statsmodels 0.9.0

**Comment by tgsmith61591** [comment]
Hrmm... you appear you be correct. I'll dig into this and see if I can figure out what may be going on. If you're right, this is a big problem especially for `out_of_sample` usage

**Comment by animalmutch** [comment]
Thanks. Glad it's not just me! I noticed that the model's endog does not get updated (which, from a quick glance at the code, it appears it should), so perhaps this is a good place to start looking.

**Comment by tgsmith61591** [comment]
You're right, it definitely appears that it *should* be working, but I suspect there is some statsmodels intricacy in the way we're updating the data that is not working as expected. [This is the line in question](https://github.com/tgsmith61591/pmdarima/blob/master/pmdarima/arima/arima.py#L689), for any would-be watchers.

Thanks again for raising this issue!

**Comment by hnagib** [comment]
> Thanks. Glad it's not just me! I noticed that the model's endog does not get updated (which, from a quick glance at the code, it appears it should), so perhaps this is a good place to start looking.

I just spent 3 hours or so trying to figure this out. Super glad I ran into this.

**Comment by tgsmith61591** [comment]
It's only on seasonal Arimas. I've found where in the statsmodels code this is affected; still working on a fix.

**Comment by tgsmith61591** [comment]
Take a look at #109 if you want to see what the proposed fix looks like... this ended up being a mess of dealing with statsmodels internals, so the actual behavior of this method has changed to allow us to support it.

**Comment by tgsmith61591** [comment]
Solved in #109. Will be releasing v1.1.1 patch shortly.
